pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -321,6 +349,90 @@ func (r *ReconcileCnsFileAccessConfig) Reconcile(request reconcile.Request) (rec
 	return reconcile.Result{}, nil
 }
 
+func (r *ReconcileCnsFileAccessConfig) configureFileVolumeACL(ctx context.Context, volumeID string, vm *vmoperatortypes.VirtualMachine, instance *cnsfileaccessconfigv1alpha1.CnsFileAccessConfig) error {
+	log := logger.GetLogger(ctx)
+	networkProvider, err := util.GetNetworkProvider(ctx)
```

Comment:
shouldnt this be `cnsoperatorutil`? There's a discrepancy between here and the package imported!

Reply:
Yes, there was duplicate package in the imports. I have addressed this in the latest commit. Please take a look